src/main/java/Storage.java

```diff
@@ -0,0 +1,131 @@
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Scanner;
+
```

Comment:
"Every class should be part of some package", says Java coding standard. It is optional but I feel you can try it out if you feel like it. (Because tbh I think you has already achieved all other requirements of Java coding standard.)